### DISCUSSION TITLE
phil files

### DISCUSSION THREAD

**Discussion by elena-pascal** [dials/dials#1578] [Q&A]
Tried to find what they are in [documentation](https://dials.diamond.ac.uk/documentation/data_files.html) but no luck. 
Wondering if it would be worth adding a link to [cctbx documentation](http://cctbx.sourceforge.net/libtbx_phil.html).

**Comment by dagewa** [comment]
Hmm, I think we could write better user-facing documentation.

**Comment by dagewa** [comment]
Nevertheless, the cctbx documentation is better than nothing, so I won't complain if you add a link :+1:

**Comment by elena-pascal** [comment]
These are also used in the documentation right?
